data/loader.js

```diff
@@ -1,4 +1,4 @@
-(async function() {
+(async function () {
```

Comment:
Why insert a space between the `function` and the `()` (same with catch statements)? AFAIK this isn't commonly formatted like this. Websites as big as archive.org don't even format like this
![{18271A49-B88F-4806-A81C-E0CEA3662366}](https://github.com/user-attachments/assets/06fe9d66-38bd-4be3-b81a-8034e9f8f517)